students/nartowska_karina/lesson_13_objects_and_classes/selective_shallow_compare.py

```diff
@@ -0,0 +1,27 @@
+class Animal:
+    def __init__(self, type, name, age):
+        self.type = type
+        self.name = name
+        self.age = age
+
+
+def compare_objects(obj_1, obj_2, atr_names):
+    atr_obj_1 = []
+    atr_obj_2 = []
+    for j in atr_names:
```

Comment:
instead of comparing two lists it could be done comparing each attribute elements so the program will exit when first attribute will be different without iterating over all attributes and creating a list of them. It can stay as it is now 😄